### PR TITLE
Sentient Cheese Fix

### LINKED ISSUE #61702: You can use sentient mobs to cheeze megafauna
Reporting client version: 514.1566

## Round ID:
[170247](https://scrubby.melonmesa.com/round/170247)

## Testmerges:
- [species changes part 1: lizards](https://www.github.com/tgstation/tgstation/pull/61440)
- [Adds a second Detective slot.](https://www.github.com/tgstation/tgstation/pull/61400)
- [Disable lobby music until sound preferences are re-added to the top bar menu.](https://www.github.com/tgstation/tgstation/pull/61467)
- [tgui: Flex basis should be zero if grow is used](https://www.github.com/tgstation/tgstation/pull/61499)

## Reproduction:

1.make a sentient fauna
2. sick it at the megafauna
3. the megafauna will not aggro on fauna
4. win

### PR DESCRIPTION
**Pull request by 567Turtle**
I made it so Megafauna and other hostile foes will fight sentient animals.

## About The Pull Request

When making a corgi or whatsoever sentient before, the megafauna, or any other hostile, wouldn't attack them. This _should_ fix it.

## Why It's Good For The Game

Fixes #61702

## Changelog

:cl: 567Turtle
fix: Made it so megafauna and other hostile mobs will attack sentient simple mobs.
/:cl:

### CODE REVIEW

**Comment by dondonpow** [comment]
I was hoping this PR fix not being able to turn cheese wheels into sentient creatures. Sad!

**Comment by 567Turtle** [comment]
> I was hoping this PR would fix not being able to turn cheese wheels into sentient creatures. Sad!

im sorry for the let down

File: code/modules/research/xenobiology/xenobiology.dm [hunk 1]
```diff
@@ -716,6 +716,7 @@
 			to_chat(SM, span_userdanger("You also become depressingly aware that you are not a real creature, but instead a holoform. Your existence is limited to the parameters of the holodeck."))
 		to_chat(user, span_notice("[SM] accepts [src] and suddenly becomes attentive and aware. It worked!"))
 		SM.copy_languages(user)
+		SM.faction = list("neutral")
```
Comment (tralezab) [inline code/modules/research/xenobiology/xenobiology.dm:719]: you should include the mob's reference as a faction as well, all living mobs have a faction to themselves!

https://github.com/tgstation/tgstation/blob/c10e1e60db4bfba3e9c0ca543a24223a37a25304/code/modules/mob/living/living.dm#L1-L10

```suggestion
		SM.faction = list([REF(src)], "neutral")
```

**Review by tralezab — CHANGES_REQUESTED** [review]

**Comment by LemonInTheDark** [comment]
Doesn't this mean sentient mobs will be hated by their peers? That's a bit of a shame

**Pull request closed without merging**